src/FixedDataTableCellGroup.js

```diff
@@ -183,6 +186,7 @@ class FixedDataTableCellGroupImpl extends React.Component {
         columnGroupWidth={columnGroupWidth}
         pureRendering={pureRendering}
         isRTL={this.props.isRTL}
+        isVisible={this.props.isVisible && isHorizontallyVisible}
```

Comment:
At this component level, `props.isVisible` refers to whether the cell group is vertically visible.
This combined with `isHorizontallyVisible` determines whether the individual cell is within the viewport.